specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkVerifier.json

```diff
@@ -0,0 +1,1345 @@
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "NetworkVerifier was updated",
+            "schema": {
+              "$ref": "#/definitions/NetworkVerifier"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkVerifier_Update": {
+            "$ref": "./examples/NetworkVerifier_Update.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "NetworkVerifier"
+        ],
+        "summary": "Deletes the specific Network Verifier resource asynchronously",
+        "operationId": "NetworkVerifier_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "networkVerifierName",
+            "description": "The name of the Network Verifier resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "NetworkVerifier delete request accepted and will complete asynchronously"
+          },
+          "204": {
+            "description": "NetworkVerifier delete request succeeded"
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "NetworkVerifier_Delete": {
+            "$ref": "./examples/NetworkVerifier_Delete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AnalysisIntent": {
+      "description": "Configuration information or intent on which to do the analysis on.",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AnalysisIntentProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
```

Comment:
I think network has its own common type , see https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2022-01-01/network.json#L140, you need to unify it to avoid conflict in lintDiff